src/Workleap.DomainEventPropagation.Subscription.PullDelivery/IEventPropagationSubscriberBuilder.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Workleap.DomainEventPropagation;
+
+public interface IEventPropagationSubscriberBuilder
+{
+    IServiceCollection Services { get; }
+
+    internal void ConfigureSubscriber(Action<EventPropagationSubscriptionOptions> configure, string optionsSectionName);
```

Comment:
I doubt the `internal` access modifier was intentionally set

```suggestion
    void ConfigureSubscriber(Action<EventPropagationSubscriptionOptions> configure, string optionsSectionName);
```

Reply:
I just reviewed your code and where this is used. 

Perhaps, the `ConfigureSubscriber` method could be an `internal` extension method on the type to make it invisible to the library consumers that this method even exists.

From my understanding, it will never be called outside of the library.